docs/set-up.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[set-up]]
 == Set up the Agent
 
-To get you off the ground, we've prepared guides for setting up the Agent with a few different popular web frameworks:
+To get you off the ground, we've prepared guides for setting up the Agent with a few different popular web frameworks and technologies:
```

Comment:
This problem existed before this PR:

A link to starting the agent is buried under "other useful documentation." Do we need to explain the starting the agent page here? Maybe how it relates to the "Get started with..." content?

Reply:
Perhaps.  I agree that it is somewhat buried. I think the "starting the agent" doc page is *almost* really just reference material and that it should hopefully only need to be referred to for the minority of users. It can be linked to from each of the "getting started" guides.

I'd prefer to punt any of this to a separate change if that is okay... perhaps as part of some of the TOC shuffling mentioned above at https://github.com/elastic/apm-agent-nodejs/pull/2837#discussion_r932728937

Reply:
That's fine 👍